src/core/tests/type_prop/transpose.cpp

```diff
@@ -104,6 +122,17 @@ TEST(type_prop, transpose_arg_rank_static_dynamic_input_order_rank_dynamic_ok) {
     EXPECT_TRUE(r->get_output_partial_shape(0).same_scheme(PartialShape::dynamic(4)));
```

Comment:
Could you please use this approach in the type_prop tests?
```suggestion
    EXPECT_EQ(r->get_output_partial_shape(0), PartialShape::dynamic(4));
```

Reply:
Done.